### PR TITLE
feat(route): add 大连理工大学公共基础学院 RSS (#13982)

### LINKED ISSUE #13982: 请求增加大连理工大学公共基础学院RSS
### 类型

大学通知

### 网站地址

http://fldpj.dlut.edu.cn/index.htm

### 网站描述

该网站是大连理工大学新成立的公共基础学院的官网。

### 需要生成什么内容？

网站上比较重要的三个模块分别是[学术动态](http://fldpj.dlut.edu.cn/kxyj/xsdt.htm)、[重要通知](http://fldpj.dlut.edu.cn/index/zytz.htm)、[院务公开](http://fldpj.dlut.edu.cn/index/ywgk.htm)，希望可以分别订阅这三个模块的通知更新。

### 额外描述

谢谢您的贡献以及您为大学牲创造的便利！

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by ueiu**
## Involved Issue / 该 PR 相关 Issue

Close #13982 

## Example for the Proposed Route(s) / 路由地址示例

```routes
/dut/fldpj/xsdt
/dut/fldpj/zytz
/dut/fldpj/ywgk
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/dut/index.js [hunk 1]
```diff
@@ -38,17 +40,28 @@ module.exports = async (ctx) => {
...
-            let dateMatch = item.parent().text().match(dateRegex);
-            if (!dateMatch) {
-                dateMatch = item.parent().parent().text().match(dateRegex);
+                let dateMatch = item.parent().text().match(dateRegex);
+                if (!dateMatch) {
+                    dateMatch = item.parent().parent().text().match(dateRegex);
+                }
+
+                itemTitle = item.text() === '' ? item.next().text() : item.text();
+                itemDate = dateMatch ? dateMatch[1].replace(/年|月/g, '-') : new Date().toString();
```
Comment (TonyRL) [inline lib/v2/dut/index.js:56]: https://docs.rsshub.app/joinus/advanced/pub-date#no-date
Do not add a date if there isn't any.

Comment (ueiu) [reply lib/v2/dut/index.js:56]: 这是之前的代码了，意思是我在这次移除掉？

Comment (TonyRL) [reply lib/v2/dut/index.js:56]: Yes

Comment (ueiu) [reply lib/v2/dut/index.js]: fixed

File: lib/v2/dut/index.js [hunk 2]
```diff
@@ -38,18 +40,28 @@ module.exports = async (ctx) => {
...
+
+                let dateMatch = item.parent().text().match(dateRegex);
+                if (!dateMatch) {
+                    dateMatch = item.parent().parent().text().match(dateRegex);
+                }
 
-            let dateMatch = item.parent().text().match(dateRegex);
-            if (!dateMatch) {
-                dateMatch = item.parent().parent().text().match(dateRegex);
+                result.title = item.text() === '' ? item.next().text() : item.text();
```
[the remark this answers is not in the document]
Comment (ueiu) [reply lib/v2/dut/index.js]: fixed

File: lib/v2/dut/index.js [hunk 3]
```diff
@@ -38,18 +40,28 @@ module.exports = async (ctx) => {
         .map((item) => {
             item = $(item);
 
-            const dateRegex = /(\d{4}[-/年]\d{2}[-/月]\d{2})/;
+            const result = {
+                link: /^http/.test(item.attr('href')) ? item.attr('href') : `${rootUrl}/${item.attr('href').replace(/^[./]+/, '')}`,
```
[the remark this answers is not in the document]
Comment (ueiu) [reply lib/v2/dut/index.js]: fixed

**Pull request merged**